src/lang/english.txt

```diff
@@ -1589,7 +1615,7 @@ STR_CONFIG_SETTING_RIGHT_MOUSE_WND_CLOSE                        :Close window on
 STR_CONFIG_SETTING_RIGHT_MOUSE_WND_CLOSE_HELPTEXT               :Closes a window by right-clicking inside it. Disables the tooltip on right-click!
 
 STR_CONFIG_SETTING_AUTOSAVE                                     :Autosave: {STRING2}
-STR_CONFIG_SETTING_AUTOSAVE_HELPTEXT                            :Select interval between automatic game saves
+STR_CONFIG_SETTING_AUTOSAVE_HELPTEXT                            :Select interval between automatic game saves. If using real-time units, only unpaused play time counts toward this interval
```

Comment:
```suggestion
STR_CONFIG_SETTING_AUTOSAVE_HELPTEXT                            :Select interval between automatic game saves. If using real-time units, only unpaused play time counts toward this interval.
```

Reply:
This is no longer part of this PR now that real-time autosave has been implemented properly. I just haven't rebased this PR.